buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -56,39 +55,73 @@ def process_build_log(failed_jobs_per_flag, already_failing_jobs, log, job):
             line = line.strip()
             if line.startswith("--incompatible_") and line in INCOMPATIBLE_FLAGS:
                 failed_jobs_per_flag[line][job["id"]] = job
-        log = log[0: log.rfind("+++ Result")]
+        log = log[0 : log.rfind("+++ Result")]
 
     # If the job failed for other reasons, we add it into already failing jobs.
     if job["state"] == "failed":
         already_failing_jobs.append(job)
 
 
 def get_html_link_text(content, link):
-    return f"<a href=\"{link}\" target=\"_blank\">{content}</a>"
+    return f'<a href="{link}" target="_blank">{content}</a>'
 
 
-def construct_success_info(failed_jobs_per_flag):
+def print_flags_ready_to_flip(failed_jobs_per_flag):
     info_text = ["#### The following flags didn't break any passing jobs"]
     for flag in INCOMPATIBLE_FLAGS:
         if flag not in failed_jobs_per_flag:
             github_url = INCOMPATIBLE_FLAGS[flag]
             info_text.append(f"* **{flag}** " + get_html_link_text(":github:", github_url))
     if len(info_text) == 1:
-        return None
-    return info_text
+        return
+    print_info("flags_ready_to_flip", "success", info_text)
 
 
-def construct_warning_info(already_failing_jobs):
+def print_already_fail_jobs(already_failing_jobs):
     info_text = ["#### The following jobs already fail without incompatible flags"]
     for job in already_failing_jobs:
         link_text = get_html_link_text(job["name"], job["web_url"])
         info_text.append(f"* {link_text}")
     if len(info_text) == 1:
-        return None
-    return info_text
+        return
+    print_info("already_fail_jobs", "warning", info_text)
+
+
+def print_projects_need_to_migrate(failed_jobs_per_flag):
+    info_text = ["#### The following projects need migration"]
+    jobs_need_migration = set()
+    for _, jobs in failed_jobs_per_flag.items():
```

Comment:
Nit: simply
for jobs in failed_jobs_per_flag.values():

Reply:
Thanks!